networkx/algorithms/dag.py

```diff
@@ -392,13 +392,18 @@ def antichains(G):
 
 
 @not_implemented_for('undirected')
-def dag_longest_path(G):
+def dag_longest_path(G, weight = 'weight', default_weight = 1):
```

Comment:
Can you remove the spaces around the equal signs?  Third bullet in https://www.python.org/dev/peps/pep-0008/#other-recommendations